wagtail_transfer/views.py

```diff
@@ -197,7 +197,7 @@ def choose_page(request):
     })
 
 
-def import_missing_object_data(importer: ImportPlanner):
+def import_missing_object_data(source, base_url, importer: ImportPlanner):
```

Comment:
As in the API proxy view above, it looks like it should be able to get the base url from the source name via the settings config - maybe we should make this consistent?

Reply:
Yep that's fair. We don't need to pass this in as an arg if it's in the settings config. I'll update that today, good catch.